crates/red_knot_python_semantic/resources/mdtest/call/dunder.md

```diff
@@ -79,13 +90,32 @@ reveal_type(this_fails[0])  # revealed: Unknown
 However, the attached dunder method *can* be called if accessed directly:
 
 ```py
-# TODO: `this_fails.__getitem__` is incorrectly treated as a bound method. This
-# should be fixed with https://github.com/astral-sh/ruff/issues/16367
-# error: [too-many-positional-arguments]
-# error: [invalid-argument-type]
 reveal_type(this_fails.__getitem__(this_fails, 0))  # revealed: Unknown | str
 ```
 
+The instance-level method is also not called when the class-level method is present:
```

Comment:
This is more of a regression test for something that was broken previously, but I can also remove it again if we don't think it adds any value.